# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1272860395 25200
#      Sun May 02 21:19:55 2010 -0700
# Node ID d0e0bb2ebebb51ff4efd27ad62138ae7d2da80dd
# Parent  6e1674216b992ba66acbe8bf544263cd6487689e
Remove tabs in .txi files causing problems with pdf formatting.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,15 @@
+2010-05-02  Rik <octave@nomad.inbox5.com>
+
+        * interpreter/contrib.txi, interpreter/install.txi, 
+        interpreter/plot.txi, interpreter/sparse.txi,
+        interpreter/octave.texi: Remove tabs causing problems with
+        pdf formatting.
+
 2010-04-14  Shai Ayal  <shaiay@users.sourceforge.net>
 
 	* interpreter/plot.txi: Document keypress and mouse callback
 	properties
 
 2010-04-07  John W. Eaton  <jwe@octave.org>
 
 	* interpreter/testfun.txi: Document runtests.
diff --git a/doc/interpreter/contrib.txi b/doc/interpreter/contrib.txi
--- a/doc/interpreter/contrib.txi
+++ b/doc/interpreter/contrib.txi
@@ -23,17 +23,17 @@
 
 This chapter is dedicated to those who wish to contribute code to Octave.
 
 @menu 
 * How to Contribute::
 * General Guidelines::
 * Octave Sources (m-files)::
 * C++ Sources::
-* Other Sources::	
+* Other Sources::
 @end menu
 
 @node How to Contribute
 @section How to Contribute
 The mailing list for Octave development discussion and sending contributions is
 @email{maintainers@@octave.org}.  This concerns the development of Octave core,
 i.e., code that goes to Octave directly.  You may consider developing and
 publishing a package instead; a great place for this is the allied Octave-Forge
@@ -224,17 +224,17 @@ endfor
 
 Don't use tabs.  Tabs cause trouble.  If you are used to them, set up your editor
 so that it converts tabs to spaces.  Format function headers like this:
 
 @example
 @group
 static bool
 matches_patterns (const string_vector& patterns, int pat_idx,
-		  int num_pat, const std::string& name)
+                  int num_pat, const std::string& name)
 @end group
 @end example
 
 @noindent
 The function name should start in column 1, and multi-line argument lists should
 be aligned on the first char after the open parenthesis.  You should put a space
 after the left open parenthesis and after commas, for both function definitions
 and function calls.
diff --git a/doc/interpreter/install.txi b/doc/interpreter/install.txi
--- a/doc/interpreter/install.txi
+++ b/doc/interpreter/install.txi
@@ -395,17 +395,17 @@ that contains the file @file{libgcc.a}.
 
 @item
 Some of the Fortran subroutines may fail to compile with older versions
 of the Sun Fortran compiler.  If you get errors like
 
 @example
 @group
 zgemm.f:
-	zgemm:
+        zgemm:
 warning: unexpected parent of complex expression subtree
 zgemm.f, line 245: warning: unexpected parent of complex
   expression subtree
 warning: unexpected parent of complex expression subtree
 zgemm.f, line 304: warning: unexpected parent of complex
   expression subtree
 warning: unexpected parent of complex expression subtree
 zgemm.f, line 327: warning: unexpected parent of complex
diff --git a/doc/interpreter/octave.texi b/doc/interpreter/octave.texi
--- a/doc/interpreter/octave.texi
+++ b/doc/interpreter/octave.texi
@@ -37,17 +37,18 @@
 @end ifnottex
 @noindent
 @strong{See also:} \args\.
 @end macro
 
 @ifinfo
 @format
 START-INFO-DIR-ENTRY
-* Octave: (octave).	Interactive language for numerical computations.
+* Octave: (octave).           Interactive language for numerical computations.
+
 END-INFO-DIR-ENTRY
 @end format
 @end ifinfo
 
 @c Settings for printing on 8-1/2 by 11 inch paper:
 @c -----------------------------------------------
 
 @setchapternewpage odd
@@ -776,17 +777,17 @@ Tips and Standards
 * Documentation Tips::          Writing readable documentation strings.
 
 Contributing Guidelines
 
 * How to Contribute::
 * General Guidelines::
 * Octave Sources (m-files)::
 * C++ Sources::
-* Other Sources::	
+* Other Sources::
 
 Trouble
 
 * Actual Bugs::                 Bugs we will fix later.
 * Reporting Bugs::              
 * Bug Criteria::                
 * Bug Tracker::                   
 * Bug Reporting::               
diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -1082,17 +1082,17 @@ properties for graphics objects.  For ex
 @group
 get (0)
     @result{} ans =
        @{
          type = root
          currentfigure = [](0x0)
          children = [](0x0)
          visible = on
-			@dots{}
+         @dots{}
        @}
 @end group
 @end example
 
 @noindent
 returns a structure containing all the properties of the root figure.
 As with all functions in Octave, the structure is returned by value, so
 modifying it will not modify the internal root figure plot object.  To
diff --git a/doc/interpreter/sparse.txi b/doc/interpreter/sparse.txi
--- a/doc/interpreter/sparse.txi
+++ b/doc/interpreter/sparse.txi
@@ -103,17 +103,17 @@ this is a simplification in the code, in
 for the first or last columns.  A short example, demonstrating this in
 C is.
 
 @example
 @group
   for (j = 0; j < nc; j++)
     for (i = cidx (j); i < cidx(j+1); i++)
        printf ("non-zero element (%i,%i) is %d\n", 
-	   ridx(i), j, data(i));
+           ridx(i), j, data(i));
 @end group
 @end example
 
 A clear understanding might be had by considering an example of how the
 above applies to an example matrix.  Consider the matrix
 
 @example
 @group
